gateway under linux crashed with update not implemented

I normally run the gateway on my linux machine, and this morning it had stopped working with the following error:
```
2019-04-06 13:55:27.552 ERROR  : 
2019-04-06 13:55:27.553 ERROR  : /home/dhylands/moziot/gateway/build/webpack:/src/platform.js:117
      throw new NotImplementedError(fn);
^
2019-04-06 13:55:27.553 ERROR  : Error: Method not implemented for platform: update
    at Object.params (/home/dhylands/moziot/gateway/build/webpack:/src/platform.js:117:1)
    at Timeout../src/app.js.setInterval [as _onTimeout] (/home/dhylands/moziot/gateway/build/webpack:/src/app.js:67:1)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! webthings-gateway@0.7.0 start: `webpack --display errors-only && node build/gateway.js "-d"`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the webthings-gateway@0.7.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/dhylands/.npm/_logs/2019-04-06T20_55_27_841Z-debug.log
```

This wasn't just an error, it actually caused the gateway to exit to the prompt.

--- a/src/app.js
+++ b/src/app.js
@@ -60,7 +60,7 @@ servers.https = createHttpsServer();
 let httpsApp = null;
 
 let updaterInterval = null;
-if (process.env.NODE_ENV !== 'test') {
+if (process.env.NODE_ENV !== 'test' && platform.implemented('update')) {
   // Start the updater
   updaterInterval = setInterval(
     () => {

